csrc/scheduler/matmul_utils.cpp

```diff
@@ -39,6 +39,7 @@
 #include "utils.h"
 
 namespace nvfuser {
+namespace matmul_utils {
```

Comment:
Good idea. As a separate issue, we should think about combining or more clearly naming the difference between `mma_utils` and `matmul_utils`.